### ISSUE TITLE
v0.6.3 fails to generate screenshots / throws JS exception

### ISSUE THREAD

**Issue by foobarth** [Codeception/CodeceptJS#593]
#### What are you trying to achieve?
Get failure screenshots with CodeceptJS 0.6.3. Note: Downgrading to 0.6.2 fixes the issue, so it must be related to some commits in between.

#### What do you get instead?
No screenshots, output directory is empty, failed test always shows a javascript error/exception at the end, which might prevent the screenshot from being generated
Error: `Error: Unexpected token ')'`

> Provide console output if related. Use `--verbose` mode for more details.

```bash
 address-update - test email fields - enGB
 • I am on page "xxxxxxxxxxxxxxxxxxxxxxxxxx"
[...]
 • I see "Email addresses do not match", ".alt_confirm_email .parsley-errors-list"
Error: Unexpected token ')'
 ✖ FAILED in 1666ms
```

### Details

* CodeceptJS version: 0.6.3
* NodeJS Version: v7.10.0
* Operating System: Debian Linux (inside a docker container)
* WebDriverIO: 4.7.1 (happens with 4.8.0 also)
* Configuration file:

```js
exports.config = {
    "tests": "./xxxxxxxxxxxxxxxxxxx/tests/frontend/*_test.js",
    "timeout": 10000,
    "output": "./xxxxxxxxxxxxxxxxxxx/tests/frontend/output",
    "helpers": {
        "WebDriverIO": {
            "url": process.env.TEST_HOST || "http://xxxxxxxxxxxxxxxxxxx:8080",
            "browser": "chrome",
            "windowSize": "1024x768"
        }
    },
    "include": {},
    "bootstrap": false,
    "mocha": {
        "reporterOptions": {
            "mochaFile": "./xxxxxxxxxxxxxxxxxxx/tests/frontend/output/codecept-results.xml"
        }
    },
    "name": "frontend"
};
```

**Comment by BorisOsipov** [comment]
@foobarth please,
provide console output with --verbose mode for more details.

**Comment by foobarth** [comment]
Sure, here you go:

```bash
   [9] <within> Queued | return step result
   [9] <within> Queued | finish within block
   [9] <within> Queued | restore session
   Emitted | step.start (Within header: I see "GRUNDLEGENDE INFOSxxx")
   • Within header: I see "GRUNDLEGENDE INFOSxxx"
   [9] <within> Error | Error
   [9] <within> Starting <teardown> session
   Emitted | test.failed ([object Object])
   [9] <teardown> Queued | hook WebDriverIO._failed()
   [9] <teardown> Queued | () => done(err)
   [9] <teardown> Stopping recording promises
   [9] <teardown> Error | Error: Unexpected token ')'
Error: Unexpected token ')'
   [9] <teardown> Stopping recording promises
 ✖ FAILED in 961ms
```

At the end of the report, with node stacktrace:
```bash
  - I.amOnPage("/de/DE/register/") at Test.Scenario (webforms/tests/frontend/basic_DE_test.js:7:7)
  
  Error
      at Object.<anonymous> (/usr/local/share/.config/yarn/global/node_modules/codeceptjs/lib/helper/WebDriverIO.js:1129:51)
      at Object.exec (/usr/local/share/.config/yarn/global/node_modules/webdriverio/build/lib/helpers/safeExecute.js:28:24)
      at Object.resolve (/usr/local/share/.config/yarn/global/node_modules/webdriverio/build/lib/webdriverio.js:189:29)
      at /usr/local/share/.config/yarn/global/node_modules/webdriverio/build/lib/webdriverio.js:538:32
      at _fulfilled (/usr/local/share/.config/yarn/global/node_modules/q/q.js:854:54)
      at self.promiseDispatch.done (/usr/local/share/.config/yarn/global/node_modules/q/q.js:883:30)


  FAIL  | 15 passed, 1 failed   // 42s
   Emitted | global.result ([object Object])
Warning: non-zero exit code 1 Use --force to continue.

Aborted due to warnings.

```

**Comment by hubidu** [comment]
Thats the same Problem as #540

**Comment by APshenkin** [comment]
@foobarth Please test it with 1.0.0 and send us feedback

**Comment by foobarth** [comment]
Seems to be fixed in 1.0.0 - i'm getting screenshots again on failures. Thanks a lot!
